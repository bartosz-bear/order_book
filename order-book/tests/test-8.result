{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}, {"id": 2, "price": 15, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
