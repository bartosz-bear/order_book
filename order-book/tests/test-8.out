{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}, {"id": 2, "price": 15, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
{"buyOrderId": 1, "sellOrderId": 3, "price": 15,  "quantity": 10}
{"buyOrderId": 2, "sellOrderId": 3, "price": 15,  "quantity": 20}
